hunts/signals.py

```diff
@@ -0,0 +1,16 @@
+from .models import Hunt
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from datetime import timedelta
+
+
+@receiver(post_save, sender=Hunt, dispatch_uid="add_default_times")
```

Comment:
Sorry about the continued back and forth.

After thinking more about the bigger picture / thread with Ryan G, I think it's better to not set any defaults here for either `start_time` nor `end_time`. I don't think hunt creation time is actually very good proxy for hunt start time. Nor do I think that start_time + 3 days would be a good proxy for hunt start time.

For the progress page feature, we can do it as follows:
- If `start_time` is specified, use that as the beginning of the chart. Otherwise use the earliest puzzle creation timestamp.
- If `end_time` is specified, use `max(end_time, current_time)` as the end of the chart. Otherwise use `current_time`. 

@rgossiaux for FYI

Reply:
I don't want to hold this up too much, I think I said my piece and beyond that I trust you guys to do what you think is best. Ultimately this is pretty minor and it should be easy to change this stuff later if it becomes a problem anyway.

Your suggestion seems reasonable to me, I especially like the earliest puzzle creation timestamp fallback which seems like a good idea.

Reply:
Sounds good to me. Choosing which time to use when start/end times are unspecified can be done when computing stats + making the chart, so I'll go ahead and implement that over there.